teamcity: failed test: version-upgrade

The following tests appear to have failed on master (roachtest): acceptance/version-upgrade

You may want to check [for open issues](https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+is%3Aopen+version-upgrade).

[#992536](https://teamcity.cockroachdb.com/viewLog.html?buildId=992536):


```
acceptance/version-upgrade
--- FAIL: roachtest/acceptance/version-upgrade (64.122s)
	test.go:644,upgrade.go:405,retry.go:188,upgrade.go:399,upgrade.go:561,upgrade.go:566,acceptance.go:65: 3: dial tcp 127.0.0.1:26261: connect: connection refused





```

Please assign, take a look and update the issue accordingly.

--- a/pkg/cmd/roachtest/upgrade.go
+++ b/pkg/cmd/roachtest/upgrade.go
@@ -455,6 +455,9 @@ func runVersionUpgrade(ctx context.Context, t *test, c *cluster) {
 		binaryVersionUpgrade("v2.0.6", nodes),
 		clusterVersionUpgrade("2.0", true /* manual */),
 
+		binaryVersionUpgrade("v2.1.2", nodes),
+		clusterVersionUpgrade("2.1", true /* manual */),
+
 		binaryVersionUpgrade("HEAD", nodes),
 		clusterVersionUpgrade("", false /* manual */),
 	}

